### PR TITLE
Fix readmes and docs content

### PR DESCRIPTION
**Pull request by svollath**
- Add/fix links to docs.epinio.io
- Add Documentation paragraph with links
- Remove local "How the docs are structured"
- Fix links in remaining local doc structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +42,12 @@ not doing operations.
 Epinio is adding the needed abstractions and intelligence to allow Developers
 to use Kubernetes as a PaaS (Platform as a Service).
 
+## Documentation
+
+Find installation and user documentation at [docs.epinio.io](https://docs.epinio.io/).
+
+Find project related documentaion here in [docs](./docs).
```
Comment (jimmykarily) [inline README.md:49]:
```suggestion
Find Epinio developer documentation here in [docs](./docs).
```

**Review by jimmykarily — DISMISSED** [review]

File: docs/howtos/development.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Look at the Makefile at the root of the project to see what is available.
 
 ### Installing Epinio
 
-You can have a look at [the dedicated document](/docs/user/howtos/install.md) for cluster
+You can have a look at [the dedicated document](https://docs.epinio.io/installation/installation.html) for cluster
```
Comment (andreas-kupries) [inline docs/howtos/development.md:41]:
```suggestion
While we have a [dedicated document](https://docs.epinio.io/installation/installation.html) for cluster
```

File: docs/howtos/development.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Look at the Makefile at the root of the project to see what is available.
...
 specific instructions, but generally this should be sufficient to get you running:
```
Comment (andreas-kupries) [inline docs/howtos/development.md:42]:
```suggestion
specific instructions, the following should generally be sufficient to get you running:
```

File: README.md [hunk 4]
```diff
@@ -153,7 +92,7 @@ Epinio documentation is organised into these four quadrants in the `./docs/` fol
 
 `Epinio` uses [Github Project](https://github.com/epinio/epinio/projects/1) for tracking issues. You can also find the issues currently being worked on in the `BackLog` section.
 
-If you would like to start contributing to `Epinio`, then you can pick up any of the cards with label `good first issue`.
+Find more information at [Contributing](./CONTRIBUTING.md).
```
Comment (andreas-kupries) [inline README.md:95]:
```suggestion
Find more information in the [Contribution Guide](./CONTRIBUTING.md).
```

File: README.md [hunk 5]
```diff
@@ -64,56 +63,9 @@ to use Kubernetes as a PaaS (Platform as a Service).
...
-- [Install on K3d](./docs/user/tutorials/install_epinio_on_k3d.md) - Install K3d and then install Epinio
-- [Install on EKS](./docs/user/tutorials/install_epinio_on_eks.md) - Install Epinio in Amazon EKS clusters
-- [Install on AKS](./docs/user/tutorials/install_epinio_on_aks.md) - Install Epinio in Azure AKS clusters
-- [Install on GKE](./docs/user/tutorials/install_epinio_on_gke.md) - Install Epinio in Google GKE clusters
-- [Install on RKE2](./docs/user/tutorials/install_epinio_on_rke.md) - Install Epinio in Rancher RKE2 clusters
-
 ## Usage
 
-- [QuickStart](./docs/user/tutorials/quickstart.md) - tutorial on how to create an org and push an application.
+- [QuickStart](https://docs.epinio.io/tutorials/quickstart.html) - tutorial on how to create an org and push an application.
```
Comment (andreas-kupries) [inline README.md:68]:
```suggestion
- [QuickStart](https://docs.epinio.io/tutorials/quickstart.html) - Tutorial on how to create a namespace and push an application.
```

File: README.md (hunk 1 above)
Comment (andreas-kupries) [inline README.md:49]:
```suggestion
Project related documentation is available from the [docs](./docs) subdirectory.
```

File: README.md [hunk 6]
```diff
@@ -49,6 +42,12 @@ not doing operations.
 Epinio is adding the needed abstractions and intelligence to allow Developers
 to use Kubernetes as a PaaS (Platform as a Service).
 
+## Documentation
+
+Find installation and user documentation at [docs.epinio.io](https://docs.epinio.io/).
```
Comment (andreas-kupries) [inline README.md:47]:
```suggestion
Installation and user documentation is available at our main [docs.epinio.io](https://docs.epinio.io/) site.
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Phrasing nits

**Pull request merged**